### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by rogertinsley**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/rogertinsley/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+`['!fix', '!resolve']`
+- Something in the code needs to be reviewed and fixed before
+  the pull request is merged.
+
+`['!fixed', '!resolved', '!completed']`
+- The issue has been resolved.
+
+To add a flag simply leave a comment with your desired flag. For
+convenience we also have a dropdown menu to add these comments
+automatically.
```
Comment (rogertinsley) [inline ReadMe.md]: This needs fixed

Comment (rogertinsley) [inline ReadMe.md]: !fix this here

Comment (rogertinsley) [inline ReadMe.md]: !fixed I've fixed it

**Pull request merged**